proposal/more-math-functions.md

```diff
@@ -0,0 +1,445 @@
+# Math Module
+
+This built-in module is available from the URL `sass:math`.
+
+## Table of Contents
+* [Variables](#variables)
+  * [`e`](#e)
+  * [`pi`](#pi)
+* [Functions](#functions)
+  * [Approximation](#approximation)
+    * [`ceil()`](#ceil)
+    * [`floor()`](#floor)
+    * [`round()`](#round)
+  * [Comparison](#comparison)
+    * [`clamp()`](#clamp)
+    * [`max()`](#max)
+    * [`min()`](#min)
+    * [`percentage()`](#percentage)
+  * [Division](#division)
+    * [`div()`](#div)
+    * [`mod()`](#mod)
+  * [Exponentiation](#exponentiation)
+    * [`log()`](#log)
+    * [`pow()`](#pow)
+    * [`sqrt()`](#sqrt)
+  * [Trigonometry](#trigonometry)
+    * [Procedures](#procedures)
+      * [Verify Inputs](#verify-inputs)
+    * [`cos()`](#cos)
+    * [`sin()`](#sin)
+    * [`tan()`](#tan)
+    * [`acos()`](#acos)
+    * [`asin()`](#asin)
+    * [`atan()`](#atan)
+    * [`atan2()`](#atan2)
+      * [Edge cases](#edge-cases)
+  * [Units](#units)
+    * [`compatible()`](#compatible)
+    * [`is-unitless()`](#is-unitless)
+    * [`unit()`](#unit)
+  * [Util](#util)
+    * [`abs()`](#abs)
+    * [`hypot()`](#hypot)
+    * [`random()`](#random)
+
+## Variables
+
+### `e`
+
+```
+$e
```

Comment:
I don't think you need to include signatures for these, since they're just variable names. I'd probably include the `$` sign in the header title though.

Reply:
Done.